docs/mobile-apps/features/mobile-app-diagnostics/app-crash-logs.md

```diff
@@ -39,18 +65,21 @@ You can download the crash log as a text file to save and share with Sauce Labs
 
 :::note
 You can also download the results using the API call below and send it to your development team:
+
 ```java
 curl --compressed \
 -O https://{SAUCE_USERNAME}:{SAUCE_ACCESS_KEY}@{DATA_CENTER}.saucelabs.com/v1/rdc/jobs/{JOB_ID}/crash.log`
 ```
+
 :::
 
 ## Limitations
 
 :::note
-- Does not capture hangs and cross-platform development frameworks.
-- Emulators/Simulators are not supported.
 
+- Does not capture hangs and cross-platform development frameworks, like React Native and .NET MAUI
+- Emulators/Simulators are not supported.
+- If you have a native crash reporting (including Backtrace), this feature will override it, therefore you will not see the crash report in your existing system.
```

Comment:
If you have a native crash reporting SDK in your application (including Backtrace)